packages/api/migrations/mongo/20240305195342-add_generic_webhook_fields.ts

```diff
@@ -0,0 +1,27 @@
+import { Db, MongoClient } from 'mongodb';
```

Comment:
we don't need the migration if all these fields are optional and the new changes won't break the old records